src/tctrack/tempest_extremes/tempest_extremes.py

```diff
@@ -285,27 +328,203 @@ def __str__(self) -> str:
         return f"DetectNodesParameters(\n\t{attributes}\n)"
 
 
+@dataclass
+class StitchNodesParameters:
+    """Dataclass containing values used by the StitchNodes operation of TE.
+
+    Parameters
+    ----------
+    output_file : str | None
+        The output filename to save the tracks. "out" in TempestExtremes.
+    in_file : str | None, optional
+        Filename of the DetectNodes output file. If this and `in_list` are ``None``, it
+        will be taken from the DetectNodes parameters. "in" in TempestExtremes.
+    in_list : str | None, optional
+        File containing a list of input files to be processed together.
```

Comment:
I didn't add this to DetectNodes at present as I think it needs extra documentation specifying how to use it (see our previous discussion on the file needing to be written in 'Tempest Extremes' format.) We could potentially set up something using the `lod_to_te()` function to make this better match the current 'Pythonic' approach we have for `in_file`.

I don't mind leaving this in, but wonder if we should make it unused/unadvised for now.
We should definitely open an issue to add a nice way to use '`in_list`' as a future feature.